### PR TITLE
Updating docs

### PR DESCRIPTION
**Pull request by fmalmeida**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/scrnaseq/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/scrnaseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by fmalmeida** [comment]
I tried to change the minimum as possible, and added a little of information on cellranger (that I don't actually know if we should).

Please let me know your thoughts, if there something still missing or to be modified here

😄

File: docs/output.md [hunk 1]
```diff
@@ -73,6 +74,14 @@ For details on how to load these into R and perform further downstream analysis,
 - `alevin/txp2gene.tsv`
   - The transcriptome to gene mapping TSV file utilized by Salmon Alevin
 
+## Cellranger
+
+Cell Ranger is a set of analysis pipelines that process Chromium single cell data to align reads, generate feature-barcode matrices, perform clustering and other secondary analysis. See [Cellranger](https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger) for more information on Cellranger.
```
Comment (apeltzer) [inline docs/output.md:79]:
```suggestion
Cell Ranger is a set of analysis scripts that processes 10X Chromium single cell data to align reads, generate feature-barcode matrices, perform clustering and other secondary analysis. See [Cellranger](https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger) for more information on Cellranger.
```

File: lib/WorkflowMain.groovy [hunk 2]
```diff
@@ -19,7 +19,7 @@ class WorkflowMain {
     // Print help to screen if required
     //
     public static String help(workflow, params, log) {
-        def command = "nextflow run ${workflow.manifest.name} --input samplesheet.csv --genome GRCh37 -profile docker"
+        def command = "nextflow run ${workflow.manifest.name} --input samplesheet.csv --genome_fasta human.fasta --gtf human.gtf -profile docker -profile docker"
```
Comment (apeltzer) [inline lib/WorkflowMain.groovy:22]:
```suggestion
        def command = "nextflow run ${workflow.manifest.name} --input samplesheet.csv --genome_fasta human.fasta --gtf human.gtf -profile docker"
```

**Comment by apeltzer** [comment]
Don't think we have to mention more than this.

**Comment by apeltzer** [comment]
@nf-core-bot fix linting

**Pull request merged**